app/src/main/AndroidManifest.xml

```diff
@@ -254,6 +262,7 @@
     <activity
         android:name=".DeepLinkIntentReceiver"
         android:exported="true"
+        android:taskAffinity=""
```

Comment:
Why can't this be the default? (package-name)

Reply:
Not sure if it is related to this but currently, if an Aptoide Activity is already in background - the deep link brings the Aptoide up with the dialog on top, instead of just the dialog on top of the current context

Reply:
If you mean without taskAffinity, it needs it because otherwise the dialog will not be independent from the activity and it will always show behind the dialog. Setting the affinity to "" means that this has no affinity. I'm not sure as to why this is required though.

Alternatively, setting it to android:taskAffinity=".DeepLinkIntentReceiver" should also work fine